sshfs-manager.sh

```diff
@@ -22,8 +22,9 @@ showMessage() {
             [INVALID_COMMAND]='[SSHFS-MGR] Please run one of the available commands from the help menu (%s --help).\n'
             [NO_SERVERS]='[SSHFS-MGR] No servers were found. Add a server using the "add-server" command.\n'
             [AVAILABLE_SERVERS]='[SSHFS-MGR] Available servers:\n'
-            [USER_INPUT_SERVER_INDEX]='\n[SSHFS-MGR] Enter a number from the list: '
+            [USER_INPUT_SERVER_INDEX]='\n[SSHFS-MGR] Enter a number from the list (supported comma separated numbers): '
             [USER_INPUT_INVALID_SERVER_INDEX]='[SSHFS-MGR] Please select a valid number from the list. Exiting...\n'
+            [USER_INPUT_ILLEGAL_SERVER_INDEX]='[SSHFS-MGR] Please select a valid comma separated number number(s) from the list. Exiting...\n'
```

Comment:
`USER_INPUT_ILLEGAL_SERVER_INDEX` is not used anywhere in this file, I think it should replace `USER_INPUT_INVALID_SERVER_INDEX`.

Also, what do you think if we'd change the message into: 
> Please enter a valid number or multiple comma separated numbers from the list. Existing...\n